lib/puppet/bindings.rb

```diff
@@ -134,7 +134,8 @@ def has_name?(name)
 
     def []=(name, value)
       unless value.is_a?(Puppet::Pops::Binder::Bindings::NamedBindings) || value.is_a?(Proc)
-        raise ArgumentError, "Given value must be a NamedBindings, or a Proc producing one, got: #{value.class}."
+        #TRANSLATORS 'NameBindings' should not be translated
+        raise ArgumentError, _("Given value must be a NamedBindings, or a Proc producing one, got: %{class_name}.") % { class_name: value.class }
```

Comment:
Since `Proc` is also a Ruby class, should it be left untranslated, or does terminology like that also get translated?

Reply:
Good question.

Reply:
Seems like it's referring to the Ruby class, since it's capitalized? This is a lot of guessing at intent, though.